immuni_analytics/core/config.py

```diff
@@ -15,8 +15,9 @@
 
```

Comment:
now the variable are not ordered anymore alphabetically 🤔

Reply:
Fixed 🆗